content/docs/ops/configuration-management.md

```diff
@@ -33,8 +33,7 @@ Security tests need to be executed in the development environment where changes
 1. All configuration changes must flow through a git repository, centrally managed through GitHub, unless they contain sensitive information. In these cases, sensitive information should be stored in an S3 bucket with a proper security policy and encryption, versioned such that changes can be easily rolled back.
 1. A change is initiated and discussed as a GitHub issue in [the relevant 18F repository](https://github.com/18F).
 1. A pull request (PR) is created that addresses the change and references the created issue.
-    If there is a staging environment available for a given repository, the PR should be
-    created against the `staging` branch. Otherwise, the PR should be created against the `master` branch on the canonical repository.
+    All components should run within a staging environment where it can undergo integration testing. Exceptions should be rare and carefully documented. A proposed code change should be a PR created against the `master` branch on the canonical repository, which must be tested within the `staging` environment before merging to master and rolling to production. 
```

Comment:
Our staging and production environments both point at the master branch, so the merge to master happens before the feature lands on staging. Do you mean feature branches should be tested on develop?

It might be simpler just to delete this bullet point, since we shouldn't be using staging branches anywhere.